05-mvc-with-rails/rails-sessions-and-auth-with-devise/readme.md

```diff
@@ -0,0 +1,319 @@
+# Devise! and sessions
+
+## Learning Objectives
+- explain how sessions give state to a web application
+- explain how user authentication utilizes sessions
+- set session hash key value pairs inside of a rails application
+- implement user authentication into a web application utilizing the devise gem
```

Comment:
I'm going to start keeping a tally of every time you say "utilizing" when you could just say "using", Army McGee. :)